OpenShift/helm/nexus-repository-manager/values.yaml

```diff
@@ -149,6 +149,9 @@ deployment:
   # # Uncomment and modify this to run a command after starting the nexus container.
   postStart:
     command:    # '["/bin/sh", "-c", "ls"]'
+  preStart:
+    command:    # '["/bin/rm", "-f", "/tmp/i4jdaemon__opt_nexus_nexus-3.20.0-04_bin_nexus"]'
```

Comment:
what's this path? is there a better example path we could provide?

Reply:
That was the path used in the helpdoc for the PID lockfile. But it's ugly and untested so I'll make it a little clearer.